src/Consumer.php

```diff
@@ -74,6 +74,8 @@ public function tick(Queue $queue, array $options = [])
             return true;
         }
 
+        $this->dispatcher->dispatch('bernard.ping');
```

Comment:
It might make sense to add a specific event class, which takes the queue which is being consumed.